### PR TITLE
* BCI-593 - Address book: Should show only one icon only * BCI-600 - Chat screen: Block ui is different compare to design * BCI-597 - Chat screen --> image or video full screen: there is less space between each icon (forward, download, three dot) compare to design * BCI-578 - Chat screen: If enter bdx amount and then click the attachments, icon gets overlap over the tooltip. * BCI-572 - Chat screen: reply card ui is different compare to design * unwanted lines removed.

### PR DESCRIPTION
**Pull request by johnbradsfiled**
…Chat screen: Block ui is different compare to design * BCI-597 - Chat screen --> image or video full screen: there is less space between each icon (forward, download, three dot) compare to design * BCI-578 - Chat screen: If enter bdx amount and then click the attachments, icon gets overlap over the tooltip. * BCI-572 - Chat screen: reply card ui is different compare to design * unwanted lines removed.

### CODE REVIEW

File: BChat/Conversations/ConversationVC+Interaction.swift [hunk 1]
```diff
@@ -465,24 +465,30 @@ extension ConversationVC : InputViewDelegate, MessageCellDelegate, ContextMenuAc
                                                 inputTextView.textColor = Colors.accent
                                                 let value = ["bdxAmount": newText]
                                                 NotificationCenter.default.post(name: .bdxAmountPassingSliderViewNotification, object: value)
+                                                CustomSlideViewBool.isFromExpandAttachment = true
```
Comment (ios-wingman) [inline BChat/Conversations/ConversationVC+Interaction.swift:468]: Change the var name (  CustomSlideViewBool)

File: BChat/Onboarding/New Design/ChatSettingsVC.swift [hunk 2]
```diff
@@ -266,6 +266,7 @@ class ChatSettingsVC: BaseVC, SheetViewControllerDelegate {
     func observeNotifications() {
         NotificationCenter.default.addObserver(self, selector: #selector(identityStateDidChange(_:)), name: Notification.Name(rawValue: "kNSNotificationName_IdentityStateDidChange"), object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(otherUsersProfileDidChange(_:)), name: Notification.Name(rawValue: "kNSNotificationName_OtherUsersProfileDidChange"), object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(blockContactPopUpYesActionTapped), name: .blockContactPopUpYesActionNotification, object: nil)
```
Comment (ios-wingman) [inline BChat/Onboarding/New Design/ChatSettingsVC.swift:269]: Change the notification name

File: BChat/Conversations/ConversationVC+Interaction.swift (hunk 1 above)
Comment (johnbradsfiled) [reply BChat/Conversations/ConversationVC+Interaction.swift]: customSlideView name changed.

File: BChat/Onboarding/New Design/ChatSettingsVC.swift (hunk 2 above)
Comment (johnbradsfiled) [reply BChat/Onboarding/New Design/ChatSettingsVC.swift]: Notification center name changed

**Review by ios-wingman — APPROVED** [review]

**Pull request merged**